sdk/servicebus/service-bus/tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "node_modules",
     "./types/**/*.d.ts",
     "./samples/**/*.ts",
-    "test/perf/*",
+    "test/perf-js-libs/*",
```

Comment:
I'm not sure if I get what goes in perf-js-libs vs perf/track-<something>? Is it just archiving these files?

Reply:
More or less, yes.

### perf-js-libs
- `perf-js-libs/azure-sb-package`
- `perf-js-libs/rhea-promise`
- `perf-js-libs/service-bus-v1`
- `perf-js-libs/service-bus-v7`

### perf/track-*
- `perf/track-1` - v1
- `perf/track-2` - v7

### Description
- `perf-js-libs` do not use the framework and have the tests for send, streaming receive and batch receive.
- `perf/track-*` uses the framework for send test, and currently has the send test only.
   - Since this uses the framework, it helps with comparisons across languages.
- Framework isn't capable of handling the receive tests, so they have to be standalone tests. 

### More thoughts
- Eventually, I intend to have the standalone tests in the `perf/track-*` folder itself and thus removing the need for `perf-js-libs/service-bus-v*` folders. 
- The `perf-js-libs/azure-sb-package` can be removed as it is pretty much outdated at the moment.
- That leaves us with `perf-js-libs/rhea-promise`, which could be moved to `perf/rhea-promise` after updating the send test with the framework and migrating the standalone receive tests.
- Finally, we'll end up with `perf/track-1`, `perf/track-2` and `perf/rhea-promise`.

Reply:
#13489